### PR TITLE
Add support for App Middlewares

### PR DESCRIPTION
**Pull request by bcooksey**
See https://github.com/zapier/zapier-platform-core/pull/119 for context

### CODE REVIEW

File: docs/build/schema.md [hunk 1]
```diff
@@ -78,6 +78,8 @@ Key | Required | Type | Description
 --- | -------- | ---- | -----------
 `version` | **yes** | [/VersionSchema](#versionschema) | A version identifier for your code.
 `platformVersion` | **yes** | [/VersionSchema](#versionschema) | A version identifier for the Zapier execution environment.
+`beforeApp` | no | [/MiddlewaresSchema](#middlewaresschema) | Before the perform method is called on your app, you can modify the execution context.
+`afterApp` | no | [/MiddlewaresSchema](#middlewaresschema) | After the perform method is called on your app, you can modify the response.
```
Comment (bcooksey) [inline docs/build/schema.md]: I guess technically these could be before executing a request object, right?

Comment (BrunoBernardino) [reply docs/build/schema.md]: I think these go after calling the `afterResponse` middleware, but we should double check.

**Review by BrunoBernardino — COMMENTED** [review]
I know we run `npm run docs` before shipping a release, but if you could include that in this PR, it reduces surprises on weird built docs like we've had in the past!

**Review by BrunoBernardino — APPROVED** [review]
👍

**Pull request merged**